static/js/tab_bar.js

```diff
@@ -131,10 +131,10 @@ exports.exit_search = function () {
     }
 };
 
-exports.update_stream_name = function (new_name) {
+exports.update_stream_name = function () {
     const stream_name = $(".stream a");
     if (stream_name !== undefined) {
```

Comment:
I would do a more straightforward check for whether we're narrowed within a stream.

Reply:
Do we need to check if we're narrowed within a stream because this function is only called [here](https://github.com/zulip/zulip/blob/7d3a31cd8b22d284002d00b72580da90382506e4/static/js/subs.js#L146),
and only when we are narrowed in the stream. Just realized this now.